src/Sfa.Tl.Matching.Data/Repositories/BackLinkHistoryRepository.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.Extensions.Logging;
+using Sfa.Tl.Matching.Data.Interfaces;
+using Sfa.Tl.Matching.Domain.Models;
+
+namespace Sfa.Tl.Matching.Data.Repositories
+{
+    public class BackLinkHistoryRepository : GenericRepository<BackLinkHistory>, IBackLinkRepository
+    {
+        private readonly MatchingDbContext _dbContext;
+
+        public BackLinkHistoryRepository(ILogger<BackLinkHistoryRepository> logger, MatchingDbContext dbContext) : base(logger, dbContext)
```

Comment:
if this repo is not exposing any specific method then remove it, just used generic repo

Reply:
Done